eve/main.yml

```diff
@@ -374,6 +374,20 @@ stages:
         command: bash scripts/run-e2e-test.sh "end2end" ${E2E_IMAGE_NAME}:${E2E_IMAGE_TAG} "end2end" "default"
         workdir: build/eve/workers/end2end
         haltOnFailure: true
+    - ShellCommand: &run_iam_policies_test
+        name: Run iam policies tests
+        env:
+          <<: [ *kind-env, *e2e-env, *http-env, *secrets-env ]
+        command: bash scripts/run-e2e-test.sh "end2end" ${E2E_IMAGE_NAME}:${E2E_IMAGE_TAG} "iam-policies" "default"
+        workdir: build/eve/workers/end2end
+        haltOnFailure: true
+    - ShellCommand: &run_lint_test
```

Comment:
Can we put the `run_lint_test`right after `git_pull`, so that we don't need to execute the following if there is any lint issue? like what we did in Vault's main.yaml https://github.com/scality/Vault/blob/development/8.4/eve/main.yml#L89